leap_year.rb

```diff
@@ -0,0 +1,27 @@
+# Write a program, leap_year.rb. It will ask the user for a starting year and an ending year, and it will then print out all the leap years between them, including the starting or ending year if those are leap years. The rules for leap years are: A leap year is divisible by 4, except for years that are divisible by 100 -- those aren't leap years -- except for years that are divisible by 400, which ARE leap years.
+
+puts "Please enter starting year"
+yearStart = gets.chomp # get the year
```

Comment:
Lets use `Integer(gets.chomp)` because if we don't then the user can input a string and the program will continue. But if we do use `Integer(gets.chomp)` then the program will stop and say ` invalid value for Integer()`.